e2e/setup/bastion.go

```diff
@@ -134,10 +134,10 @@ func CreateVM(ctx context.Context, location, groupName, sshKeyData string, nic n
 				},
 				StorageProfile: &compute.StorageProfile{
 					ImageReference: &compute.ImageReference{
-						Publisher: to.StringPtr("CoreOS"),
-						Offer:     to.StringPtr("CoreOS"),
-						Sku:       to.StringPtr("Stable"),
-						Version:   to.StringPtr("2135.6.0"),
+						Publisher: to.StringPtr("kinvolk"),
+						Offer:     to.StringPtr("flatcar-container-linux-free"),
+						Sku:       to.StringPtr("stable"),
+						Version:   to.StringPtr("2512.2.1"),
```

Comment:
Why not the latest version?

Reply:
No reason, I just took the version from one of the releases. I'll change it with the latest release.